### DISCUSSION TITLE
"These are traits that are used to convert automatically between types."

### DISCUSSION THREAD

**Discussion by blueglyph** [JelteF/derive_more#355] [Q&A]
Ref: https://docs.rs/derive_more/latest/derive_more/

Is this a typo, or did you do something special so the conversions are indeed handled automatically?

As far as I know, those traits are not automatic, but some of them automatically implement another one (like `From` with `Into`); maybe that's what you meant? Automatic conversion is one of Rust's taboos.

**Comment by JelteF** [comment]
I guess it's a doc improvement. "easily" would be a better word. Feel free to create a PR.
